plugin/main.go

```diff
@@ -38,9 +38,9 @@ func resolveModeFromParameters(env *plugins.Environment) {
 		exitWithMessage(env, "unsupported parameter name")
 	}
 	switch env.Request.Parameters[0].Value {
-	case "incomp": // Base incompatibility scanning
+	case "1": // Base incompatibility scanning
```

Comment:
How will this affect the developer experience?

Reply:
just introduces ease of use, number rather than longer string, am hoping to create readme(s) to help developer experience, for these individual file reports, the readme would explain each parameter value